python/examples/extensions/Custom_Metrics.ipynb

```diff
@@ -0,0 +1,216 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "uNhjtINB2EaL"
+      },
+      "source": [
+        "# Custom Metrics\n",
+        "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/whylabs/whylogs/blob/1.0.x/python/examples/extensions/Custom_Metrics.ipynb)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "WT8XU-Df065F"
+      },
+      "outputs": [],
+      "source": [
+        "!pip install whylogs --pre"
```

Comment:
we should remove the --pre and replace "!" with "%", and also comment it, to prevent it being runned every time the notebook is being tested:
`# %pip install -q whylogs`